### PR TITLE
cleaning up tutorial hero: moving launch button and breadcrumb

### PR DESCRIPTION
**Pull request by scott-confluent**
### Description

Per discussions, we're cleaning up the hero and making the "Launch..." button more prominent.

Screen Shot 2022-04-14 at 1 26 39 PM

### CODE REVIEW

File: _includes/recipe-content-answer.html [hunk 1]
```diff
@@ -1,39 +1,28 @@
...
-       <div class="columns">
-         <div class="column is-full">
-           <h3 class="title">Launch recipe in Confluent Cloud</h3>
            {% for step in site.data.harnesses[page.static_data][page.stack].answer.steps %}
 
             <div class="tutorial-try-it-step content">
+            {% if step.title %}
              <h4 class="subtitle" id="{{ step.title | slugify }}">
-               <div class="text">{{ step.title }}</div>
+               <div class="text">{{ step.title }} Fake Title</div>
```
Comment (iPzard) [inline _includes/recipe-content-answer.html:10]: Is this ok to keep "Fake Title"?

File: assets/sass/main.scss [hunk 2]
```diff
@@ -256,6 +256,14 @@ a {
     }
   }
 
+  .container {
+    margin: 15px 0;
+
+    .title {
+      color: #E6F5FB;
```
Comment (iPzard) [inline assets/sass/main.scss:263]: No variable to use for this value?

**Review by iPzard — APPROVED** [review]

**Review by ybyzek — CHANGES_REQUESTED** [review]
Thanks for the PR.  Some comments

1. Although this `index.html` page isn't used on the live site, the hero centering has changed:

Screen Shot 2022-04-14 at 16 16 32 PM

2. Now there is a mismatch: the non-recipe tutorials have `Back to recipes` in the hero, but the recipes have that `Back to recipes` under the hero.  I also think it's weird having it below.  Can we move it back to where it was, because it looks better (this is subjective and just my opinion) and is consistent with the other pages?

**Review by ybyzek — APPROVED** [review]
Thanks for the fixes, LGTM!

**Pull request merged**